testing/adios2/interface/TestADIOSSelection.cpp

```diff
@@ -289,12 +293,14 @@ TEST_F(ADIOS2_CXX11_API_Selection, SelectionWrite)
 
     // read back
     auto arr_read = MultiArrayT(ref.dims());
-    m_IOReader.SetEngine("BP3");
+    //    m_IOReader.SetEngine("BP3");
```

Comment:
Maybe remove this

Reply:
> Question, where is this functionality implemented for BP5?
The changes are in BP5Deserializer, where a single call got replaced by an if with a vastly more complex then clause.

Also, this contains a bug fix for the BP5Serializer that was causing a memory overrun when there was more than one block/variable written and min/max was enabled.  That it got past msan means that we're not running any of those sorts of tests there.  That is somewhat worrisome as it means that some pretty basic functionality isn't being sanitized.  (I know why, we can't do everything within the hour time limit, but its probably something we should look at doing nightly, or weekly, or something.)

Reply:
> That is somewhat worrisome as it means that some pretty basic functionality isn't being sanitized

Indeed true, we have this idea of extending the sensitization test in our backlog. This discovery increases the priority in the task.